haas/rest.py

```diff
@@ -164,10 +172,19 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        if isinstance(methods, list):
```

Comment:
Agree with @shwsun - compatibility was to avoid updating everything else as well as reducing the amount of code. This is primarily used for testing anyhow.